src/test/java/org/opentripplanner/routing/fares/OrcaFareServiceTest.java

```diff
@@ -171,30 +190,31 @@ public void calculateFareTest(List<Ride> rides,
      * the values used in {@link OrcaFareServiceImpl} to determine the correct ride type.
      */
     private static List<Ride> getTrip(OrcaFareServiceImpl.RideType... rideTypes) {
-        List<Ride> rides = new ArrayList<>();
-        for (OrcaFareServiceImpl.RideType rideType : rideTypes) {
-            rides.add(tripStrategy.get(rideType));
-        }
-        return rides;
+        return Arrays.stream(rideTypes).map(tripStrategy::get).collect(Collectors.toList());
     }
 
-    private static Ride getRide(String agencyId) {
-        return getRide(agencyId, "-1", -1, null);
+    private static Ride getRide(String agencyId, long durationMins) {
```

Comment:
In light of https://github.com/ibi-group/OpenTripPlanner/pull/58/commits/6360ad63b7b69d5b6a7a4c03ad3c3a478e5eb14d#r670777222, the duration argument probably needs to mean "duration since the last free-transfer boarding" or something.